code4romania/teacher-workout-ios
# Implement forgot password UI

### What does it fix?


Closes #3 



Implemented forgot password flow UI. 
**Note** For sending the email to the API, we are going to create a separate issue.

### How has it been tested?





1. Run the app
2. Select "Autentifica-te"
3. Select "Ti-ai uitat parola?"
4. Check forgot password screen is presented
5. Type a valid email address and tap "Reseteaza"
6. Check confirmation screen is presented

![Simulator Screen Shot - iPhone 12 Pro Max - 2021-08-18 at 00 28 23](https://user-images.githubusercontent.com/481425/129803628-29dee3c4-ac22-4d99-8bac-8bd8a3dc6806.png)
![Simulator Screen Shot - iPhone 12 Pro Max - 2021-08-18 at 00 28 31](https://user-images.githubusercontent.com/481425/129803631-38e84cf0-bfb7-41c0-bd96-900494f4511f.png)
